src/System.Runtime/tests/Performance/Perf.StringBuilder.cs

```diff
@@ -144,5 +144,47 @@ public static void StringBuilderToString()
                 }
             }
         }
+
+        [Benchmark]
+        [InlineData(20)]
+        [InlineData(200)]
+        [InlineData(1000)]
+        public void AppendMemoryAsMemory(int length)
+        {
+            PerfUtils utils = new PerfUtils();
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                // Setup - Create a string of the specified length
+                string builtString = utils.CreateString(length);
+                ReadOnlyMemory<char> memory = builtString.AsMemory();
+                StringBuilder empty = new StringBuilder();
+
+                // Actual perf testing
+                using (iteration.StartMeasurement())
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
```

Comment:
You need to define the inner iteration count value by decorating the Benchmark attribute:
`[Benchmark(InnerIterationCount=100_000)]`

See https://github.com/Microsoft/xunit-performance/blob/master/README.md#authoring-benchmarks.

Also, can you please share the new perf test results with these changes?

Reply:
Done, results below.